target/start-mailserver.sh

```diff
@@ -663,11 +664,8 @@ function _setup_dovecot_quota() {
 		    echo -n >/tmp/docker-mailserver/dovecot-quotas.cf
```

Comment:
This can overwrite configured quotas, right? Is it perhaps better to use touch?

Reply:
btw: `echo -n` is not needed. `>/tmp/docker-mailserver/dovecot-quotas.cf` is enough.
Should we replace all `echo -n >`?